cmd/spire-server/cli/run/run.go

```diff
@@ -385,6 +385,7 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 	}
 
 	logger, err := log.NewLogger(logOptions...)
+	sc.DefaultLogLevel, _ = logrus.ParseLevel(c.Server.LogLevel)
```

Comment:
log.NewLogger returns a `*logrus.Logger`. The `Level` field should already be set based on the logOptions.
```suggestion
	sc.DefaultLogLevel = logger.Level
```

Reply:
The default (launch) level should not be set off the dynamically changeable log level.

Reply:
What I'm saying is that if you query the log level after the logger is initialized, and before anything can touch it, the level at that moment will be the launch level that was provided in the options. It just avoids you having to parse the log level again.

Reply:
IRL decided to leave it.